pkg/binding/controller.go

```diff
@@ -222,9 +221,6 @@ func (c *Controller) run(ctx context.Context, workers int) error {
 						c.handleObject(new)
 					},
 					DeleteFunc: func(obj interface{}) {
-						if shouldSkipDelete(obj) {
-							return
-						}
 						c.handleObject(obj)
```

Comment:
What happens if obj is a `cache.DeletedFinalStateUnknown` ?

Reply:
This was previously only checking if the object is a BindingPolicy. I think this should be issued as a bug and handled separately.

Reply:
Then go ahead and create an Issue.

Reply:
Opened issue #1792